src/app/dspace/authorization/components/login-form.component.ts

```diff
@@ -0,0 +1,142 @@
+import {Component} from 'angular2/core';
+import {NgForm} from 'angular2/common';
+import {TranslateService, TranslatePipe} from "ng2-translate/ng2-translate";
+
+import {AuthorizationService} from '../services/authorization.service';
+
+import {FormModal, ModalAction} from '../../../utilities/components/form-modal.component';
+
+/**
+ *
+ */
+@Component({
```

Comment:
Seems like the translate pipe is not used in this component to take care of i18n